Gemfile.lock

```diff
@@ -7,10 +7,15 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
+    addressable (2.7.0)
```

Comment:
Were these changes added manually? I don't see any changes made to the `Gemfile`.

Reply:
There are different ways of managing dependencies for gems. The "bundler way" is to list all dependencies in the gemspec then call `gemspec` in the Gemfile to load dependencies listed there.

https://bundler.io/v2.0/guides/creating_gem.html#testing-our-gem

I don't know much about this project, but [hoe](https://www.zenspider.com/projects/hoe.html) has another way of managing gems.